src/views/manage-identities/ManageIdentities.tsx

```diff
@@ -0,0 +1,403 @@
+                  },
+                ];
+              });
+            });
+        });
+    });
+  }, [accountEvm, deployedTokens, setOwnTokens]);
+
+  useEffect(() => {
+    if (registry) {
+      const unsubAgentsAdded: WatchContractEventReturnType =
+        watchIdentityRegistryAgentAddedEvent(
+          {
+            onLogs: (data) => {
+              setAddedAgents(((prev: any) => {
+                return [...prev, ...data.map((log: any) => log.args[0])];
+              }) as any);
+            },
+          },
+          registry as `0x${string}`,
+        );
+      return () => {
+        unsubAgentsAdded();
+      };
+    }
+  }, [registry]);
+
+  useEffect(() => {
+    if (addedAgents?.length && registry) {
+      const unsubAgentsRemoved: WatchContractEventReturnType =
+        watchIdentityRegistryAgentRemovedEvent(
+          {
+            onLogs: (data) => {
+              const agents = data.map((log: any) => log.args[0]);
+
+              if (agents.find((agent) => addedAgents.includes(agent))) {
+                setAddedAgents((prev) =>
+                  prev.filter((agent) => !agents.includes(agent)),
+                );
+              }
+            },
+          },
+          registry as `0x${string}`,
+        );
+
+      return () => {
+        unsubAgentsRemoved();
+      };
+    }
+  }, [addedAgents, registry]);
+
+  return (
+    <>
+      <Flex gap="4" pt="4" className="modify-identities-container">
+        <style>
+          {`
+          .modify-identities-container {
+            flex-direction: flex;
+          }
+          .modify-identities-container-section {
+            width: 49%;
+          }
+          @media screen and (max-width: 1200px) {
+            .modify-identities-container {
+              flex-direction: column;
+            }
+            .modify-identities-container-section {
+              width: 80%;
+            }
+          }
+        `}
+        </style>
+        <Flex
+          direction="column"
+          className="modify-identities-container-section"
+        >
+          <Box width="62%">
+            <MenuSelect
+              data={ownTokens.map((tok) => ({
+                value: tok.address,
+                label: tok.name,
+              }))}
+              label="Select tokens to manage identities"
+              onTokenSelect={handleTokenSelect}
+            />
+          </Box>
+          <Box mt="4" width="62%">
+            <MenuSelect
+              data={registryAgents.map((identity) => ({
+                value: identity,
+                label: identity,
+              }))}
+              label="Select identity wallet to manage"
+              onTokenSelect={handleIdentitySelect}
+            />
+          </Box>
+          <Flex direction="column" mt="9">
+            <Text fontWeight="bold" style={{ fontSize: 14 }}>
+              Selected identity wallet: {selectedIdentity}
+            </Text>
+            {selectedIdentity !== "0x" ? (
```

Comment:
what is this "0x" value? can we have a proper `undefined` or empty string instead?

Reply:
this would also help us with condition to drop the empty <></> thing and have only

selectedIdentity && ...